packages/next/src/withTRPC.tsx

```diff
@@ -102,7 +110,8 @@ export function withTRPC<
         }
 
         const config = getClientConfig({});
-        const queryClient = new QueryClient(config.queryClientConfig);
+        const queryClient =
+          config.queryClient ?? new QueryClient(config.queryClientConfig);
```

Comment:
just for my understanding: there's a second mention of `new QueryClient` in this file:

https://github.com/trpc/trpc/blob/09f41cda414ca9615e783da0ce9bffee81fc6b2e/packages/next/src/withTRPC.tsx#L183

wouldn't we need to make that change there as well?

Reply:
It's also in a few other places, like the ssg helpers.